src/org/gk/gkCurator/SynchronizationDialog.java

```diff
@@ -191,18 +191,14 @@ private void clearDeleteRecord() {
 		if (deleteList != null) {
 			java.util.List list = deleteList.getSelection();
 			if (list.size() > 0) {
-				try {
```

Comment:
Why remove the try/catch?

Reply:
The try/catch was removed because the method call wrapped in the try block does not throw the exception being caught.

Reply:
Have you pushed the changes in XMLFileAdaptor into GitHub?

Reply:
Yes, it is part of this PR: https://github.com/reactome/CuratorTool/pull/14/commits/c72b0ec0f41a31f33c81045eee5f8f72fd4772d8#diff-fbb43a391912c3c0eb5a8816ac6c5cfcR1106